test/isolated/config.test.js

```diff
@@ -85,4 +85,19 @@ describe('config', () => {
     )
     expect(config.target).toBe('serverless')
   })
+
+  it('Should throw when next.config.ts is there', () => {
```

Comment:
I think there should be another test case here, which ensures that Next does not throw when both `next.config.js` and `next.config.ts` exist.